.vs/ProjectSettings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "CurrentProjectSetting": "x64-Debug (default)"
```

Comment:
Maybe the `.vs` folder was committed accidentally 👀?

Reply:
Ah, yes. Please add a .gitignore entry for this and remove the files from these commits.